crates/clvm-traits/docs/derive_macros.md

```diff
@@ -50,11 +50,11 @@ let ptr = value.to_clvm(a).unwrap();
 assert_eq!(Tiers::from_clvm(a, ptr).unwrap(), value);
 ```
 
-### Solution
+### List
 
-The solution representation is the same as list, except it does not check the nil terminator when parsing.
-This allows it to be lenient to additional parameters that are in the CLVM object, since they don't affect anything.
-If you want your solution to be parsed strictly, you can use list instead.
+The list representation is the same as proper list, except it does not check the nil terminator when parsing.
+This allows it to be lenient to additional parameters that are in the CLVM object, which is especially useful
+for preventing errors when parsing on-chain solutions and output conditions, where you wouldn't want strictness.
```

Comment:
it might be good to explain that consensus rules are relaxed and the reason is to leave the door open for future soft-forks